### ISSUE TITLE
Rule Change: (max-len > ignorePattern)

### ISSUE THREAD

**Issue by dbads** [eslint/eslint#16463]
### What rule do you want to change?

ignorePattern option of max-len rule should accept multiple patterns. right now it accepts only a string

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

Other

### Example code

```js
NA
```

### What does the rule currently do for this code?

NA

### What will the rule do after it's changed?

NA

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @dbads, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates) so we are not adding new options or changing the existing options in the max-len rule.

You can use `|` regex syntax to merge multiple patterns into a single pattern: `"ignorePattern": "pattern1|pattern2|pattern3"`.
